submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,79 @@
+from sprint_b.helpers import FileManager, Vocabulary, EpithetGenerator
+from flask import json
+import os
+import json
+import shutil
+
+"""     use pytest      """
```

Comment:
You've used multi-line comment notation to write a single line comment to yourself to remind yourself to run test with pytest?